### PR TITLE
edit wwf

### PR DESCRIPTION
**Pull request by yeah800**
*Please edit everything below that's UPPERCASE and starts with a "📝"*:

**What change is this pull request introducing?**
📝CHANGES

### CODE REVIEW

File: data.js [hunk 1]
```diff
@@ -74,29 +73,53 @@ exports.data = [
             label: "wwf",
         },
         description: {
-            content: "The WWF is active since 1961 and therefore a well experienced player within the environmental engagement game. The biggest Pro on WWF is that it is well known and People trust the organization. They do not have a precise focus on the climate crises but there is also a focus on it. One of their goals for 2030 is to reduce greenhouse gas emissions by 40%.",
+            content: "The WWF is active since 1961 and therefore a well experienced player within the environmental engagement game. The biggest pro of WWF is that it is well known and people trust the organization. They do not have a precise focus on the climate crises but there is also a focus on it. One of their goals for 2030 is to reduce greenhouse gas emissions by 40%.",
```
Comment (reasn) [inline data.js]: 1. Would shorten it and make it more neutral/descriptive: `The WWF is well known and people trust it`.
2. Can you add a source for the trust statement?
3. `They do not have a precise focus on the climate crises but there is also a focus on it.` would kill this sentence or substantiate it
4. `One of their goals for 2030 is to reduce greenhouse gas emissions by 40%.` is this a goal or a demand? How are they working towards it?

File: data.js [hunk 2]
```diff
@@ -74,29 +73,53 @@ exports.data = [
...
             updatedBy: "Andre", updatedAt: "2019-08-18"
         },
         established: {
             content: "[1961](http://wwf.panda.org/knowledge_hub/history/)",
             updatedBy: "Andre", updatedAt: "2019-08-06"
         },
         kpis: {
-            content: "- Actions done / pa \n- Participants in actions\n- Members\n- Press Mentions",
+            content: "- Actions done / pa \n- Participants in actions\n- Members\n- Press mentions \n- advised organization \n- Visitors on Website",
```
Comment (reasn) [inline data.js]: 1. Web traffic can be cheaply bought, I wouldn't use it as a KPI.
2. imo `/ pa` is redundant, every KPI can be measured in absolute and relative numbers.
-> "- Successful actions\n- Participants in actions\n- Members\n- Press mentions \n- advised organization"

File: data.js [hunk 3]
```diff
@@ -74,29 +73,53 @@ exports.data = [
...
             updatedBy: "Andre", updatedAt: "2019-08-06",
-
         },
+        actions: {
+            content: "These vary. With their expertise as a big environmental player they advise (countries, communities and private companies)[https://www.worldwildlife.org/initiatives/climate] in the topic."+
```
Comment (reasn) [inline data.js]: `These vary. ` -> `The big variety of actions is [available online](TODO LINK).\n- They advise (countries, communities and private companies)[https://www.worldwildlife.org/initiatives/climate] on many environmental topics.`

File: data.js [hunk 4]
```diff
@@ -74,29 +73,53 @@ exports.data = [
...
+                    "Own (press releases)[https://www.worldwildlife.org/press-releases?initiative_id=climate] are done."+
```
Comment (reasn) [inline data.js]: I wouldn't consider press releases noteworthy actions

File: data.js [hunk 5]
```diff
@@ -74,29 +73,53 @@ exports.data = [
...
+                    "Research like (Climate Crowd)[https://www.wwfclimatecrowd.org/], which is trying to crowdsource information on the impact of climate change."+
+                    "Another focus to fight climate crisis is their (deforstation program.)[https://www.worldwildlife.org/initiatives/climate]",
```
Comment (reasn) [inline data.js]: `... fight the climate crisis...`
`deforestation`
`.` to the end of the sentence

File: data.js [hunk 6]
```diff
@@ -74,29 +73,53 @@ exports.data = [
...
+            updatedBy: "Andre", updatedAt: "2020-01-14",
+
         activeCountries: {
             content: "[about 100 countries](http://wwf.panda.org/knowledge_hub/history/50_years_of_achievements/)",
             updatedBy: "Andre", updatedAt: "2019-08-18",
         },
+        participation: {
+            content: "In their most reaching action the (World Hour)[https://www.worldwildlife.org/pages/earth-hour] over 12,000 landmarks and monuments switched off their lights for a whole hour and over 3 Billion social media posts had been done in about (this action in 2017)[http://awsassets.panda.org/downloads/Earth_Hour_Report_2017.pdf#page=6]",
```
Comment (reasn) [inline data.js]: wording, would do `"Their action with the highest social media reach, the [World Hour](https://www.worldwildlife.org/pages/earth-hour), had over 12,000 landmarks and monuments switch off their lights for an hour. In 2017 this resulted in over [3 billion social media posts](http://awsassets.panda.org/downloads/Earth_Hour_Report_2017.pdf#page=6)"`

File: data.js [hunk 7]
```diff
@@ -74,29 +73,53 @@ exports.data = [
...
             updatedBy: "Andre", updatedAt: "2019-08-18"
         },
+
+        employees: {
+            content: "6,200 in (2015)[https://web.archive.org/web/20150206122603/http://wwf.panda.org/wwf_quick_facts.cfm]",
+            updatedBy: "Andre", updatedAt: "2019-08-06"
     },
-    {
+    members: {
+        content: "5+ M in (2015)[https://web.archive.org/web/20150206122603/http://wwf.panda.org/wwf_quick_facts.cfm]",
```
Comment (reasn) [inline data.js]: I find `5M+` easier to read, wdyt?

File: data.js [hunk 8]
```diff
@@ -74,29 +73,53 @@ exports.data = [
...
+        updatedBy: "Andre", updatedAt: "2020-01-14"
+    },
+        pressArticles: {
+            content: "They have a [press section](https://www.worldwildlife.org/press-mentions?year=2019) for external press.",
```
Comment (reasn) [inline data.js]: "Their [press mentions section](https://www.worldwildlife.org/press-mentions?year=2019) is actively maintained."

**Review by reasn — CHANGES_REQUESTED** [review]

File: data.js (hunk 1 above)
Comment (yeah800) [reply data.js]: restructured. left 4 like it is

File: data.js (hunk 2 above)
Comment (yeah800) [reply data.js]: done

File: data.js (hunk 4 above)
Comment (yeah800) [reply data.js]: As it is with the reports of greenpeace media is using these reports.
I also would not considers reports as they were not from wwf or greenpeace

File: data.js (hunk 3 above)
Comment (yeah800) [reply data.js]: changed the wording

File: data.js (hunk 5 above)
Comment (yeah800) [reply data.js]: changed the wording

File: data.js (hunk 6 above)
Comment (yeah800) [reply data.js]: check. looks better

File: data.js (hunk 7 above)
Comment (yeah800) [reply data.js]: okidoki

**Review by yeah800 — COMMENTED** [review]
all done. take a look

File: data.js (hunk 4 above)
Comment (reasn) [reply data.js]: Not sure I understand these two sentences. Press releases are intended to be used by the media. But creating them in my opinion is not a noteworthy action. With comprehensive studies that would be different. But what I see there are "just" press releases (as the name also indicates).

**Review by reasn — CHANGES_REQUESTED** [review]
One last thing around the press releases

File: data.js (hunk 8 above)
Comment (yeah800) [reply data.js]: should had been done also on the 22.01...

File: data.js [hunk 9]
```diff
@@ -74,29 +73,55 @@ exports.data = [
...
         kpis: {
-            content: "- Actions done / pa \n- Participants in actions\n- Members\n- Press Mentions",
-            updatedBy: "Andre", updatedAt: "2019-08-06",
-
+            content: "- Actions done  \n- Participants in actions\n- Members\n- Press mentions \n- Advised organization",
+            updatedBy: "Andre", updatedAt: "2020-01-21",
         },
+        actions: {
+            content: "There is a big variety. Here are some examples:"+
+                    "With their expertise as a big environmental player they advise (countries, communities and private companies)[https://www.worldwildlife.org/initiatives/climate] in the topic."+
```
Comment (reasn) [inline data.js:89]: Needs a space in "examples:With"

Comment (reasn) [reply data.js:89]: Same issues in the next 5 lines. Also newlines `\n`s are missing.

File: data.js [hunk 10]
```diff
@@ -74,29 +73,55 @@ exports.data = [
...
             updatedBy: "Andre", updatedAt: "2019-08-18"
         },
+
+        employees: {
+            content: "6,200 in (2015)[https://web.archive.org/web/20150206122603/http://wwf.panda.org/wwf_quick_facts.cfm]",
+            updatedBy: "Andre", updatedAt: "2019-08-06"
     },
-    {
+    members: {
+        content: "5M+ in (2015)[https://web.archive.org/web/20150206122603/http://wwf.panda.org/wwf_quick_facts.cfm]",
```
Comment (reasn) [inline data.js:115]: wrong brackets for links

File: data.js [hunk 11]
```diff
@@ -74,29 +73,55 @@ exports.data = [
...
+            updatedBy: "Andre", updatedAt: "2020-01-22"
+        },
         yearlyIncome: {
             content: "- 752M € in [2018](http://d2ouvy59p0dg6k.cloudfront.net/downloads/wwfintl_annualreview20182.pdf)" +
                 "\n- 145.9M € in [2013](http://d2ouvy59p0dg6k.cloudfront.net/downloads/ar2013_v15_lr_1.pdf)",
             updatedBy: "Andre", updatedAt: "2019-08-18"
         },
+
+        employees: {
+            content: "6,200 in (2015)[https://web.archive.org/web/20150206122603/http://wwf.panda.org/wwf_quick_facts.cfm]",
```
Comment (reasn) [inline data.js:111]: wrong brackets for links

**Review by reasn — CHANGES_REQUESTED** [review]

File: data.js (hunk 9 above)
Comment (yeah800) [reply data.js:89]: done

File: data.js (hunk 10 above)
Comment (yeah800) [reply data.js:115]: done

File: data.js (hunk 11 above)
Comment (yeah800) [reply data.js:111]: done

**Comment by yeah800** [comment]
worked through all suggestions & fixed some other brackets

**Pull request closed without merging**